Gem: RobotecWarehouseBarriersAndCagesAssets

MaciejFilipow opened the pull request:
+Bollards - Bollard01, Bollard02;
+CCTVCameras - CCTVCamera01, CCTVCamera02;
+LightCurtains - LightCurtains;
+ModularFences - High/Low;

MaciejFilipow committed:
Gem: RobotecWarehouseMiscellaneous

+Bollards - Bollard01, Bollard02;
+CCTVCameras - CCTVCamera01, CCTVCamera02;
+LightCurtains - LightCurtains;
+ModularFences - High/Low;

knicked commented:
I was able to successfully load the assets into an o3de project and they look good to me

<img width="1893" height="1080" alt="image" src="https://github.com/user-attachments/assets/ad482935-de5d-4afb-8adc-4e5844e55c94" />

MaciejFilipow committed:
Asset: FireExtinguisher

+FireExtinguisher01 asset, prefab,textures

pawelbudziszewski reviewed (changes requested):
- [x] `Bollard01` doesn't look like a typical warehouse asset. Move it to a new gem: `RobotecFencesAssets` in here: https://github.com/RobotecAI/robotec-generic-assets 
- [x] Move `Bollard02` to https://github.com/RobotecAI/robotec-warehouse-assets/tree/main/
- [x] Move `CCTVCameras` to a new gem `RobotecSurveillanceCameras` in here: https://github.com/RobotecAI/robotec-generic-assets
- [x] Move `FireExtinguishers` to here: https://github.com/RobotecAI/robotec-generic-assets/tree/main/RobotecGenericDecorationsAssets/Assets
- [x] Move `LightCurtains` to here: https://github.com/RobotecAI/robotec-warehouse-assets/tree/main/RobotecWarehouseBarriersAndCagesAssets
- [x] These `Fences` are not warehouse-specific as well. Move them to `RobotecFencesAssets` in here: https://github.com/RobotecAI/robotec-generic-assets 
cc @pijaro, @jhanca-robotecai

MaciejFilipow committed:
Remove RobotecWarehouseMiscellaneous gem, moving assets

- Remove RobotecWarehouseMiscellaneous gem
- Move LightCurtain, Bollard02 assets to RobotecWarehouseBarriersAndCagesAssets Gem

pawelbudziszewski reviewed (approved):
Everything seems to be fine now
